src/models/order.js

```diff
@@ -0,0 +1,29 @@
+const { Schema, model, SchemaTypes } = require("mongoose");
+
+const schema = new Schema({
+    productionDay: {
+        type: Date,
+        required: true,
+        unique: true,
```

Comment:
Are you sure this should be unique? So there will be no other ordres on that day ?